### ISSUE TITLE
Wrong answers from Longformer model even on simple questions

### ISSUE THREAD

**Issue by danishpruthi** [huggingface/transformers#5650]
I am using the pretrained model `allenai/longformer-large-4096-finetuned-triviaqa`, however upon inspecting it on my system and the demo on the huggingface website, the outputs seem off even for very simple examples and samples from the dataset.

1. [Example 1](https://www.dropbox.com/s/9h3dcqpwq0n1b05/download%20%283%29.png?dl=0)
2. [Example 2](https://www.dropbox.com/s/40e93m2odix8x1p/download%20%284%29.png?dl=0)
3. [Example 3](https://www.dropbox.com/s/s5t1k6jluyzfs33/download%20%286%29.png?dl=0)
4. [Example 4](https://www.dropbox.com/s/oyps5a5gr2e4c25/download%20%287%29.png?dl=0)

Other models for QA (like `bert-large-uncased-whole-word-masking`) get such simple examples right.

**Comment by patrickvonplaten** [comment]
Yeah this is related to a bug, see: https://github.com/huggingface/transformers/pull/4615

cc @mfuntowicz @julien-c - we should refactor the squad preprocessing in pipelines to make longformer work.

**Comment by danishpruthi** [comment]
Hi @patrickvonplaten: are there are any updates with respect to this?

**Comment by patrickvonplaten** [comment]
We will probably start working on a fix in ~2 weeks.
